src/reducers/messagesReducer.js

```diff
@@ -0,0 +1,13 @@
+import * as types from '../actions/actionTypes';
+import initialState from './initialState';
+
+export default function messagesReducer(state = initialState.messages, action) {
+  switch (action.type) {
+    case types.FETCH_MESSAGES_SUCCESS:
+      return [...state, action.payload];
```

Comment:
I would have checked on the key / id of a message like done [here](https://redux.js.org/docs/basics/ExampleTodoList.html#reducers) to avoid duplicates.

Can be useful: https://redux.js.org/docs/basics/ExampleTodoList.html#reducers